docs/user-guide/basic-usage.md

```diff
@@ -39,12 +39,60 @@ spec:
           serviceName: myserviceb
           servicePort: 80
 ```
-
+ 
 When you apply this yaml, 2 ingress resources will be created managed by the **ingress-nginx** instance. Nginx is configured to automatically discover all ingress with the `kubernetes.io/ingress.class: "nginx"` annotation.
 Please note that the ingress resource should be placed inside the same namespace of the backend resource.
 
+If cluster version >= 1.19 the Ingress resource above will not work.
+You deploy the following so that it match the new api version.
+This will require changes to the Ingress objects as shown below, and a new object from type "kind: IngressClass"
+```
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-myservicea
+spec:
+  rules:
+  - host: myservicea.foo.org
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: myservicea
+            port: 
+              number: 80
+  ingressClassName: nlb
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-myserviceb
+spec:
+  rules:
+  - host: myserviceb.foo.org
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: myserviceb
+            port: 
+              number: 80
+  ingressClassName: nlb
+---
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  name: nlb
```

Comment:
Why nlb? Can't the name also be nginx?

Reply:
Updated, also updated controller to match the latest from controller code